types/gitignore-parser/gitignore-parser-tests.ts

```diff
@@ -0,0 +1,15 @@
+import gitignore = require('gitignore-parser');
+
+const gitCompile = gitignore.compile('.git');
+
+const isAccept: boolean = gitCompile.accepts('test');
+const isDeny: boolean = gitCompile.maybe('test');
+const isMaybe: boolean = gitCompile.denies('test');
+
+const parseVal = gitignore.parse('.git');
```

Comment:
adding a typedef to parseVal like `isAccept`, `isDeny`, etc ... to ensure typechecks will catch typedef changes down the line